tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true,
+    "noEmit": false,
```

Comment:
What does it mean?

Reply:
Actually that is necessary for test enviroment, is used for no emit the output [doc](https://www.typescriptlang.org/tsconfig#noEmit)... 
**_Note_**: when run ````yarn test``` the test enviroment rewrite that config.